src/translations.typ

```diff
@@ -19,6 +19,16 @@
     "definition": "Definition",
     "proof": "Beweis"
   ),
+  "fr": (
+    "theorem": "Théorème",
+    "lemma": "Lemme",
+    "corollary": "Corollaire",
+    "remark": "Remarque",
+    "proposition": "Proposition",
+    "example": "Exemple",
+    "definition": "Définition",
+    "proof": "Preuve"
+  ),
```

Comment:
"it":  (
    "theorem":  "Teorema",
    "lemma":  "Lemma",
    "corollary":  "Corollario",
    "remark": "Osservazione",
    "proposition": "Proposizione",
    "example": "Esempio",
    "definition": "Definizione",
    "proof": "Dimostrazione"
),

Reply:
This is the Italian version

Reply:
You should make a pull request instead: fork the repository, create a new branch, add italian translations and make a pull request.

Reply:
Yes that would be the best way, but I can also add them for you in case you don't want to deal with that.